### PR TITLE
backup: add json table field for analyze

### PR DESCRIPTION
**Pull request by 3pointer**
part of https://github.com/pingcap/br/issues/598

### CODE REVIEW

File: proto/backup.proto [hunk 1]
```diff
@@ -72,6 +72,10 @@ message Schema {
     uint64 total_bytes = 5;
 
     uint32 tiflash_replicas = 6;
+
+    // json_table represents dump stats for a table, which generate by DumpStatsToJSON
+    // https://github.com/pingcap/tidb/blob/e136429d8dc5d70f43cd3f94179b0b9f47595097/statistics/handle/dump.go#L116
+    bytes json_table = 7;
```
Comment (kennytm) [inline proto/backup.proto:78]: JSONTable is just the name of the type. The name of the variable should be "stats".

Comment (3pointer) [reply proto/backup.proto]: Done, PTAL~

**Review by kennytm — COMMENTED** [review]
LGTM

**Comment by 3pointer** [comment]
`CMakeFiles/ssl.dir/build.make:254: recipe for target 'CMakeFiles/ssl.dir/src/ssl/handshake.cc.o' failed` 🤔️

**Comment by glorv** [comment]
LGTM

**Review by glorv — APPROVED** [review]

**Review by kennytm — APPROVED** [review]

**Review by overvenus — APPROVED** [review]
LGTM

**Pull request merged**